### ISSUE TITLE
KeyError: key not found: isolator_threshold

### ISSUE THREAD

**Issue by vassilevsky** [palkan/isolator#14]
Hello :)

We've added Isolator to our app and are getting the following error on a regular basis:

```
KeyError: key not found: isolator_threshold
  from isolator/ext/thread_fetch.rb:7:in `raise'
  from isolator/ext/thread_fetch.rb:7:in `fetch'
  from isolator.rb:67:in `transactions_threshold'
  from isolator.rb:91:in `within_transaction?'
  from isolator/adapters/base.rb:31:in `notify?'
  from isolator/adapters/base.rb:26:in `notify'
  from isolator/adapter_builder.rb:24:in `block (2 levels) in add_patch_method'
  from sidekiq/client.rb:74:in `push'
  from sidekiq/client.rb:131:in `push'
  from sidekiq/scheduled.rb:26:in `block (2 levels) in enqueue_jobs'
  from sidekiq/scheduled.rb:15:in `each'
  from sidekiq/scheduled.rb:15:in `block in enqueue_jobs'
  from sidekiq.rb:96:in `block in redis'
  from connection_pool.rb:64:in `block (2 levels) in with'
  from connection_pool.rb:63:in `handle_interrupt'
  from connection_pool.rb:63:in `block in with'
  from connection_pool.rb:60:in `handle_interrupt'
  from connection_pool.rb:60:in `with'
  from sidekiq.rb:93:in `redis'
  from sidekiq/scheduled.rb:14:in `enqueue_jobs'
  from sidekiq/scheduled.rb:77:in `enqueue'
  from sidekiq/scheduled.rb:68:in `block in start'
  from sidekiq/util.rb:16:in `watchdog'
  from sidekiq/util.rb:25:in `block in safe_thread'
```

We use:
* Ruby 2.5.0
* Sidekiq 5.1.1
* Isolator 0.2.1

Please advise.

**Comment by palkan** [comment]
Hi!

Thanks for the report, first of all.

I've push a fix to master, please, try it (and if it solves the problem I release a new version)

**Comment by vassilevsky** [comment]
I can confirm that the fix (13bb3ab637f797a3f61db758864beab290340b88) is working. And it should — the default value makes fetching very stable :) Thanks!

**Comment by palkan** [comment]
Released as 0.2.2
